src/channel/index.ts

```diff
@@ -0,0 +1,1007 @@
+  }
+
+  /**
+ * Get channel id
+ *
+ */
+  id (): string {
+    const id = channelId.get(this)
+    if (id != null) {
+      return id
+    } else throw new ChannelError('Channel is not initialized')
+  }
+
+  /**
+ * Get channel's fsm id
+ *
+ */
+  fsmId (): string {
+    const id = channelFsmId.get(this)
+    if (id != null) {
+      return id
+    } else throw new ChannelError('Channel is not initialized')
+  }
+
+  /**
+ * Trigger a transfer update
+ *
+ * The transfer update is moving tokens from one channel account to another.
+ * The update is a change to be applied on top of the latest state.
+ *
+ * Sender and receiver are the channel parties. Both the initiator and responder
+ * can take those roles. Any public key outside of the channel is considered invalid.
+ *
+ * @param from - Sender's public address
+ * @param to - Receiver's public address
+ * @param amount - Transaction amount
+ * @param sign - Function which verifies and signs offchain transaction
+ * @param metadata
+
+ * @example channel.update(
+ *   'ak_Y1NRjHuoc3CGMYMvCmdHSBpJsMDR6Ra2t5zjhRcbtMeXXLpLH',
+ *   'ak_V6an1xhec1xVaAhLuak7QoEbi6t7w5hEtYWp9bMKaJ19i6A9E',
+ *   10,
+ *   async (tx) => await account.signTransaction(tx)
+ * ).then(({ accepted, signedTx }) =>
+ *   if (accepted) {
+ *     console.log('Update has been accepted')
+ *   }
+ * )
+ */
+  async update (
+    from: string,
+    to: string,
+    amount: number | BigNumber,
+    sign: SignTx,
+    metadata: string[] = []): Promise<{
+      accepted: boolean
+      signedTx?: string
+      errorCode?: number
+      errorMessage?: string
+    }> {
+    return await new Promise((resolve, reject) => {
+      enqueueAction(
+        this,
+        (_channel, state) => state?.handler === handlers.channelOpen,
+        (channel, _state) => {
+          send(channel, {
+            jsonrpc: '2.0',
+            method: 'channels.update.new',
+            params: { from, to, amount, meta: metadata }
+          })
+          return {
+            handler: handlers.awaitingOffChainTx,
+            state: {
+              resolve,
+              reject,
+              sign
+            }
+          }
+        }
+      )
+    })
+  }
+
+  /**
+ * Get proof of inclusion
+ *
+ * If a certain address of an account or a contract is not found
+ * in the state tree - the response is an error.
+ *
+ * @param addresses
+ * @param addresses.accounts - List of account addresses to include in poi
+ * @param addresses.contracts - List of contract addresses to include in poi
+ * @example channel.poi({
+ *   accounts: [
+ *     'ak_Y1NRjHuoc3CGMYMvCmdHSBpJsMDR6Ra2t5zjhRcbtMeXXLpLH',
+ *     'ak_V6an1xhec1xVaAhLuak7QoEbi6t7w5hEtYWp9bMKaJ19i6A9E'
+ *   ],
+ *   contracts: ['ct_2dCUAWYZdrWfACz3a2faJeKVTVrfDYxCQHCqAt5zM15f3u2UfA']
+ * }).then(poi => console.log(poi))
+ */
+  async poi <Prefix extends EncodingType>(
+    { accounts, contracts }: {
+      accounts: string[]
+      contracts?: string[] }): Promise<EncodedData<Prefix>> {
```

Comment:
```suggestion
      contracts?: string[] }): Promise<EncodedData<'pi'>> {
```
🤷‍♀️ , also I would keep the return type on a separate line